src/wp-admin/site-editor.php

```diff
@@ -118,6 +118,10 @@ static function ( $classes ) {
 	),
 );
 
+if ( ! empty( $_GET['postId'] ) ) {
+	$preload_paths[] = add_query_arg( 'context', 'edit', rest_get_route_for_post( $_GET['postId'] ) );
```

Comment:
This is lacking any sort of sanitization.

Also `rest_get_route_for_post` can return an empty string if no post is found, in which case one wouldn't want do add this to the array.

Reply:
Yes, the empty string check has already been added in GB.

Reply:
Should be fixed. Is `get_post` enough sanitisation?